src/DevSoc.Eventathon/ClientApp/src/forms/SignupForm.js

```diff
@@ -19,7 +26,7 @@ export default function SignupForm() {
                    type="password"
                    {...register("password", { required: true, minLength: 8 })}
             />
-            {errors.Password && <p> Password must contain more than seven characters</p>}
+{/*            {errors.password && <p> Password must contain more than seven characters</p>}*/}
```

Comment:
Is this meant to be commented out?

Reply:
Yeah if I remember correctly this doesn't compile otherwise and I couldn't figure out why.